src/components/motor_output_reordering/body.html

```diff
@@ -0,0 +1,31 @@
+<div class="motorOutputReorderComponent">
+    <h3 i18n="motorsRemapDialogTitle" class="motorOutputReorderComponentHeader"></h3>
+    <div class="componentContent" id="dialogMotorOutputReorderMainContent">
+        <canvas id="motorOutputReorderCanvas"></canvas>
+        </canvas>
```

Comment:
Two closing canvas. I suppose one of them must be removed.

Reply:
how this was even working lol

Reply:
The browser is "smart" and removes a lot of HTML code when we add strange things 😋